features/admin.authentication-flow-builder.v1/providers/authentication-flow-provider.tsx

```diff
@@ -162,6 +167,16 @@ const AuthenticationFlowProvider = (props: PropsWithChildren<AuthenticationFlowP
         }
     }, []);
 
+    /**
+     * On initialAuthenticationSequence change,
+     * if the authentication sequence passed is not empty, set it to the state.
+     */
+    useEffect(() => {
```

Comment:
Hope you conducted a regressions round and validated the flows manually.
 - Adding an conditional auth template from the side panel and checking if it's working as expected.
 - etc.

Reply:
Yes,
I tested the following flows manually,

- Adding predefined basic auth template and update application.
- Adding predefined conditional auth template and update application.
- Reverting to default.
- Configure custom login flow using visual editor and update application.
- Testing above three flows in console settings.

Reply:
ack